lib/modules/versioning/types.ts

```diff
@@ -33,6 +33,8 @@ export interface VersioningApi {
   matches(version: string, range: string | Range): boolean;
 
   valueToVersion?(version: string): string;
+
+  subset?(sub: string, dom: string): boolean;
```

Comment:
please add a tsdoc to describe usage, so other community contributors know what `subset` on versioning means.

Reply:
Done.